### PR TITLE
Jasper Report Update and Html Report Image Embedding

### PR DESCRIPTION
**Pull request by Gothami**

### CODE REVIEW

**Comment by cnapagoda** [comment]
Reviewed, Please merge.

**Comment by cnapagoda** [comment]
@Nipuni @manoj-kumara @Kishanthan Reviewed, Please merge.

**Comment by Gothami** [comment]
This PR is for [1]
[1] https://wso2.org/jira/browse/CCOMMONS-3

File: components/reporting/org.wso2.carbon.reporting.util/src/main/java/org/wso2/carbon/reporting/util/types/HtmlReport.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2009, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2009-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Nipuni) [inline components/reporting/org.wso2.carbon.reporting.util/src/main/java/org/wso2/carbon/reporting/util/types/HtmlReport.java]: please update the year to 2016

File: components/reporting/org.wso2.carbon.reporting.util/src/main/java/org/wso2/carbon/reporting/util/types/HtmlReport.java [hunk 2]
```diff
@@ -47,26 +52,30 @@ public ByteArrayOutputStream generateHtmlReport(JasperPrint jasperPrint)
...
+            jasperPrint.setProperty("net.sf.jasperreports.export.html.exclude.origin.band.2", "pageFooter");
+            final Map<String, String> images = new HashMap<>();
+            HtmlExporter exporterHTML = new HtmlExporter();
+            //setting up an input stream to HtmlExporter object
+            SimpleExporterInput exporterInput = new SimpleExporterInput(jasperPrint);
+            exporterHTML.setExporterInput(exporterInput);
+            //setting up an output stream to HtmlExporter object
+            SimpleHtmlExporterOutput simpleHtmlExporterOutput = new SimpleHtmlExporterOutput(outputStream);
+            exporterHTML.setExporterOutput(simpleHtmlExporterOutput);
+	        //To generate a html report we need to embed images using base64 encoding.
```
Comment (Nipuni) [inline components/reporting/org.wso2.carbon.reporting.util/src/main/java/org/wso2/carbon/reporting/util/types/HtmlReport.java]: formatting issue in the comment

**Pull request merged**